test/util.go

```diff
@@ -4,20 +4,51 @@ import (
 	"context"
 	"fmt"
 	"strings"
+	"time"
 
 	testcluster "github.com/mesosphere/ksphere-testing-framework/pkg/cluster"
 	networkutils "github.com/mesosphere/ksphere-testing-framework/pkg/utils/networking"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	timeTimeout = 300 * time.Second
```

Comment:
Please add a comment that talks about why this was chosen so that the next person that might need to edit this could have some idea why it was chosen.